docs/Deploying/TUF-Update-Guide.md

```diff
@@ -0,0 +1,188 @@
+# TUF Update Guide
+
+This document is a walkthrough guide for how a Fleet member can become a publisher of updates for [Fleet's TUF service](tuf.fleetctl.com).
+The roles needed to push new updates are `targets`, `snapshot` and `timestamp`.
+
+## Security
+
+- TUF keys for `targets`, `snapshot` and `timestamp` should be stored on a USB stick.
+- The keys are stored encrypted with a passphrase stored in 1Password.
+
+## Sync Fleet's TUF repository
+
+For simplicity we sync the repository to the USB stick.
+```sh
+cd /Volumes/FLEET-TUF/repository
+
+cd /Volumes/FLEET-TUF
+mkdir -p repository
+mkdir -p keys
+mkdir -p staged
+
+aws s3 sync s3://fleet-tuf-repo ./repository
+```
+
+## Generate targets+snapshot+timestamp keys
+
+```sh
+tuf gen-key targets
+Enter targets keys passphrase:
+Repeat targets keys passphrase:
+Generated targets key with ID ae943cb8be8a849b37c66ed46bdd7e905ba3118c0c051a6ee3cd30625855a076
+```
+```sh
+tuf gen-key snapshot
+Enter snapshot keys passphrase:
+Repeat snapshot keys passphrase:
+Generated snapshot key with ID 1a4d9beb826d1ff4e036d757cfcd6e36d0f041e58d25f99ef3a20ae3f8dd71e3
+```
+```sh
+tuf gen-key timestamp
+Enter timestamp keys passphrase:
+Repeat timestamp keys passphrase:
+Generated timestamp key with ID d940df08b59b12c30f95622a05cc40164b78a11dd7d408395ee4f79773331b30
+```
+
```

Comment:
The passphrase for these keys should be stored in 1Password, correct? Do we have any guidelines on whether that should be a private vault or a shared vault in Fleet's 1Password?

Reply:
> The passphrase for these keys should be stored in 1Password, correct?
Correct. (see second bullet of `Security`).

> Do we have any guidelines on whether that should be a private vault or a shared vault in Fleet's 1Password?
No guidelines yet. Currently on a private vault.